packages/map/lib/src/leaflet/classes/leaflet-map.spec.ts

```diff
@@ -222,12 +222,12 @@ describe('The leaflet map', () => {
 		});
 
 		it('should find the current location', () => {
-			map.locate();
+			map.locate(16);
```

Comment:
Not sure why you set `16` as a parameter.
Since it's an optional parameter, Is it necessary here?

Reply:
It is necessary since the locate() function will always take an argument. It's an optional parameter in the component ( `@Input() public zoomLevel = 19;` )

Reply:
✅ 
Thanks for the clarification!